.pre-commit-config.yaml

```diff
@@ -0,0 +1,40 @@
+repos:
+  - repo: https://github.com/asottile/pyupgrade
+    rev: v3.2.2
+    hooks:
+      - id: pyupgrade
```

Comment:
pyupgrade is a formatter that upgrades to newer Python syntax: https://github.com/asottile/pyupgrade